cmd/storj/cmd/rb.go

```diff
@@ -32,26 +33,56 @@ func init() {
 func deleteBucket(cmd *cobra.Command, args []string) error {
 	ctx := process.Ctx(cmd)
 
+	identity, err := rbCfg.Load()
+	if err != nil {
+		return err
+	}
+
+	bs, err := rbCfg.GetBucketStore(ctx, identity)
+	if err != nil {
+		return err
+	}
+
 	if len(args) == 0 {
-		return errs.New("No bucket specified for deletion")
+		fmt.Println("No bucket specified for deletion")
+		return nil
+	}
+
+	u, err := utils.ParseURL(args[0])
+	if err != nil {
+		return err
 	}
 
-	so, err := getStorjObjects(ctx, rbCfg)
+	_, err = bs.Get(ctx, u.Host)
```

Comment:
Same here  - we should check if `u.Host` is empty before using it.

Reply:
Done